lib/linguist/heuristics.yml

```diff
@@ -714,6 +714,24 @@ disambiguations:
   - language: Markdown
     # Markdown syntax for scdoc
     pattern: '^#+\s+(NAME|SYNOPSIS|DESCRIPTION)'
+- extensions: ['.scm']
+  rules:
+  - language: Scheme
+    pattern:
+    - '(?:''[\(\w\*]|[\w\-]+->[\w\-]+|\b\.\.\.\b|\([+\-#:<>\/=~\)]|~>)'
+    - '^#:\w+'
+    - '#\w*\('
+    - '^\s*\((?i:define|import|library|let)'
+    negative_pattern:
+    - '(["_\]\)] (@\w|\?)|'
+    - '\(#[\w-]+[!\?]'
```

Comment:
This regex is incomplete. Looks like it's missing a closing `)`.

Reply:
Is there a bundle command to do more verbose compatibility checks? I'm trying to see if there's a more granular way to run the heuristic tests to address the failures.

Reply:
> Is there a bundle command to do more verbose compatibility checks? I'm trying to see if there's a more granular way to run the heuristic tests to address the failures.

I'm not sure what level of granularity you're looking for, but you can run the test file directly with: `bundle exec ruby test/test_heuristics.rb` or `bundle exec rake test TEST=test/test_heuristics.rb`. You can add `-v` to make things a bit more verbose.

You can run a specific test in that file with: `bundle exec ruby test/test_heuristics.rb --name test_1_by_heuristics` or `bundle exec rake test TEST=test/test_heuristics.rb TESTOPTS="--name=test_1_by_heuristics"`. You can add `-v` to `TESTOPTS` to make things a bit more verbose.